src/Swarm/Game/World.hs

```diff
@@ -214,6 +222,14 @@ newWorld f = World f M.empty M.empty
 emptyWorld :: t -> World t e
 emptyWorld t = newWorld (WF $ const (t, Nothing))
 
+lookupCosmoTerrain ::
```

Comment:
If we've renamed the type `Cosmo` to `Cosmic` I think we should also rename any functions that have the word `Cosmo` in them.